polkadot/xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -65,20 +71,31 @@ benchmarks! {
 				max_weight: Weight::MAX,
 			},
 			// Worst case is looking through all holdings for every asset explicitly - respecting the limit `MAX_ITEMS_IN_ASSETS`.
```

Comment:
```suggestion
			// Worst case is looking through all holdings for every asset explicitly - respecting 
			// the limit `MAX_ITEMS_IN_ASSETS`.
```

fmt (see CI errors)

Reply:
Oh so the fmt changed this comment and the CI doesnt like it ? (from what i understand from the error in the CI)

Reply:
nvm its the other way around then, the CI wants this to be shorter but i guess the default FMT config isnt cutting it

Reply:
As you can see from CI, you can run `cargo +nightly fmt --all -- --check` to fix it